linkedList.js

```diff
@@ -4,5 +4,116 @@
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
 function linkedListGenerator(){
+  var head = null;
+  var tail = null;
+  var length = 0;
+  // var node = {
+  //           value: null,
+  //           next: {},
+  //         };
+  var linkedList = {
+    getHead: function(){
+              return head;
+            },
+    getTail: function(){
+              var thisNode = head;
+              if(!thisNode){
+                return null;
+              }
+              while(thisNode.next !== null){
+                thisNode = thisNode.next;
+              }
+              return thisNode;
+            },
+    add: function(value){
+            var newNode = {
+                  value: value,
+                  next: null
+                  };
+            if(head === null && tail === null){
```

Comment:
Does it matter if `tail` is `null`?

Also, consider the fact that you are always running 
```javascript
tail = newNode;
```
in both blocks. It is the same thing if you were to move that line _after_ the `else` block.

```
if (today === 'tuesday') {
  // Take out the trash
  // Go to work
} else {
  // Go to work
}
```
is the same as 
```
if (today === 'tuesday') {
  // Take out the trash
}
// Go to work
```